content/applications/finance/accounting/receivables/customer_invoices/electronic_invoicing.rst

```diff
@@ -0,0 +1,33 @@
+====================
+Electronic invoicing
+====================
+
+EDI, or electronic data interchange, allows to rapidly send information digitally using standard
+formats.
```

Comment:
longer explanation maybe, to explain what it means. If I don't know EDI yet, I can't understand from your intro that it means the invoices data is included.